helpers/bench.nix

```diff
@@ -0,0 +1,93 @@
+{ snabbSrc ? (builtins.fetchTarball https://github.com/snabbco/snabb/tarball/next)
+# specify how many times is each benchmark ran
+, numTimesRunBenchmark ? 20
+# specify on what hardware will the benchmarks be ran
+, requiredSystemFeatures ? [ "performance" ]
+, SNABB_PCI0 ? "0000:01:00.0"
+, SNABB_PCI_INTEL0 ? "0000:01:00.0"
+, SNABB_PCI_INTEL1 ? "0000:01:00.1"
```

Comment:
Tricky, do doable. Once we have https://github.com/snabblab/snabblab-nixos/issues/33 we'll have to use `impureEnvVars` to allow these variables in chroots.